object-search/README.md

```diff
@@ -38,37 +36,23 @@ The code can of course run natively on your local machine, please [read the Jina
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**
 
-- [TL;DR: Just Show Me the Pokemon!](#tldr-just-show-me-the-pokemon)
 - [Download and Extract Data](#download-and-extract-data)
 - [Index Image Data](#index-image-data)
-- [Query Top-K Visually Similar Images](#query-top-k-visually-similar-images)
-- [Build Docker Image](#build-docker-image)
+- [Start the server](#start-the-server)
+- [Query via REST API](#query-via-rest-api)
 - [Troubleshooting](#troubleshooting)
 - [Documentation](#documentation)
 - [Community](#community)
 - [License](#license)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-
-## TL;DR: Just Show Me the Pokemon!
-
-> *I want Pokémon! I don't care about Jina cloud-native neural search or whatever big names you throw around, just show me the Pokémon!*
-
-We have a pre-built Docker image ready to use:
-
-```bash
-docker run -p 34567:34567 -e "JINA_PORT=34567" jinaai/hub.app.bitsearch-pokedex search
-```
-
-Then you can `curl`/query/js it via HTTP POST request. [Details here](#query-via-rest-api).
-
 ## Download and Extract Data
 ### Use Flickr8k
 Although the model is trained on Flickr30k, you can test on Flickr8k dataset, which is a much smaller version of 
```

Comment:
It is not valid that the model has been ttained in f30k, we should emphasize that it is not the best dataset for the job and this kind of stuff. That they are entitled to use any other dataset